cve_json_schema/v5.x_discuss/cve513.schema

```diff
@@ -592,6 +602,35 @@
                   "type": "string",
                   "description": "Description of the vulnerability. Eg., [PROBLEMTYPE] in [COMPONENT] in [VENDOR] [PRODUCT] [VERSION] allows [ATTACKER] to [IMPACT] via [VECTOR]. OR [COMPONENT] in [VENDOR] [PRODUCT] [VERSION] [ROOT CAUSE], which allows [ATTACKER] to [IMPACT] via [VECTOR].",
```

Comment:
I guess this should now explicitly note that it should be a "**Plain text** description of the vulnerability..."?

Reply:
added it.